[workers] airtable tutorial revision (#2959)
cloudflare/cloudflare-docs: products/workers/src/content/tutorials/handle-form-submissions-with-airtable/index.md

While the majority of this function is concerned with parsing the request body (the data being sent as part of the request), there are two important things to note. First, if the HTTP method sent to this function is not `POST`, you will return a new response with the status code of [`405 Method Not Allowed`](https://httpstatuses.com/405). 

The variable `reqBody` represents a collection of fields, which are key-value pairs for each column in your Airtable table. By formatting `reqBody` as an object with a collection of fields, you are creating a new record in our table with a value for each field.

> After you call `createAirtableRecord` (the function you will define next), you will redirect the client back to our `FORM_URL`. This function can be changed, for example, to redirect to a Thank You page, or something similar.

Reviewer: ```suggestion
After you call `createAirtableRecord` (the function you will define next), you will redirect the client back to `FORM_URL`. This function can be changed, for example, to redirect to a Thank You page, or something similar.
```

```

You will notice that your function is deployed to a unique URL — for example, `https://workers-airtable-form.cloudflare.workers.dev`. This represents the first part of your front-end form's `action` attribute — the second part is the path for your form handler, which is `/submit`. In your front-end UI, configure your `form` tag as seen below:

```html
<form action="https://workers-airtable-form.cloudflare.workers.dev/submit" method="POST" class="...">
<!-- The rest of your HTML form -->
```

After you have deployed your new form (refer to the [HTML forms](https://developers.cloudflare.com/pages/tutorials/forms) tutorial if you need help creating a form), you should be able to submit a new form submission, and see the value show up immediately in Airtable:

![Example GIF of complete Airtable and serverless function integration](./example.gif)

## Conclusion

With this tutorial completed, you have created a Workers serverless function that can accept form submissions and persist them to Airtable. You have learned how to parse form data, set up environment variables, and use the `fetch` API to make requests to external services outside of our Workers function.

> You can find the source for this project — both the front-end UI, as well as the serverless function that communicates with Airtable, [on GitHub](https://github.com/cloudflare/workers-airtable-form).

Reviewer: ```suggestion
You can find the source for this project — both the front-end UI and the serverless function that communicates with Airtable — [on GitHub](https://github.com/cloudflare/workers-airtable-form).
```
